docs/rule/no-aria-unsupported-elements.md

```diff
@@ -0,0 +1,38 @@
+# no-aria-unsupported-elements
+
+Certain reserved DOM elements like `meta`, `html`, `script`, `style` do not support ARIA roles, states and properties. This is often because they are not visible. This rule enforces that these DOM elements do not contain the `role` and/or `aria-*` props.
```

Comment:
Can we turn this list of elements into a bullet-point list here? That has nicer readability, especially in case the list becomes longer.

Reply:
Cleaned up the wording a bit. Thanks!